syntax = "proto3";

package chatDemo;

service ChatService
{
    rpc ClientSide(stream Connect) returns (stream Connect){}
}

message Connect
{
    string text = 1;
    int32 id = 2;
}

